contrib/devtools/README.md

```diff
@@ -90,6 +90,20 @@ example:
 BUILDDIR=$PWD/build contrib/devtools/gen-manpages.sh
 ```
 
+gen-bitcoin-conf.sh
+===================
+
+Generates a bitcoin.conf file in `share/examples/` by parsing the output from `bitcoind -help`. This script is run during the
```

Comment:
Nit. I realize --help and -help does the same thing with the current version of bitcoind..

```suggestion
Generates a bitcoin.conf file in `share/examples/` by parsing the output from `bitcoind --help`. This script is run during the
```

Reply:
done!